pdf/core/crypt.go

```diff
@@ -1022,6 +1113,189 @@ func (crypt *PdfCrypt) Encrypt(obj PdfObject, parentObjNum, parentGenNum int64)
 	return nil
 }
 
+var ivAESZero = make([]byte, aes.BlockSize)
```

Comment:
Is this thread safe? Not that the library is thread safe at the moment, but may be in the future.
Could put into PdfCrypt